pt-BR/generators.md

```diff
@@ -1,20 +1,20 @@
 **DO NOT READ THIS FILE ON GITHUB, GUIDES ARE PUBLISHED ON https://guides.rubyonrails.org.**
 
-Creating and Customizing Rails Generators & Templates
+Criando e Personalizando Geradores e Modelos de Rails
```

Comment:
Acho que template poderia ficar só em itálico pra não confundir com os `Models`, talvez algo como:

 ```suggestion
Criando e Personalizando Geradores Rails e *Templates*
```

O que acha, fica muito fora?